### ISSUE TITLE
RabbitMQ: Disk Free Space and GC Bytes Reclaimed outside of the PATROL parameter range

### ISSUE THREAD

**Issue by bertysentry** [sentrysoftware/studioX-templates#19]
In the RabbitMQ template, some monitored values are in bytes. These values may be greater than 2^31, causing the below error message:
```
[WARN] RabbitMQ - GC Bytes Reclaimed: The value (2656520304) is outside the PATROL parameter range [2147483646, -2147483647].
```

We need to update the Numeric Value Extractions to convert bytes to MB or GB (something reasonable).

**Comment by bertysentry** [comment]
@RazeemM Could you please have a look? Thanks!

**Comment by bertysentry** [comment]
Solved with PR #22
